### PR TITLE
K8SPXC-339 fix output in installation examples (mysql, haproxy)

### PR DESCRIPTION
**Pull request by fiowro**
[K8SPXC-339](https://jira.percona.com/browse/K8SPXC-339)

### CODE REVIEW

File: source/gke.rst [hunk 1]
```diff
@@ -168,19 +168,19 @@ Now run ``mysql`` tool in the percona-client command shell using the password ob
 
 .. code:: bash
 
-   mysql -h cluster1-proxysql -uroot -proot_password
+   mysql -h cluster1-proxysql-0 -uroot -proot_password
```
Comment (delgod) [inline source/gke.rst:171]: no, lets connect via service
```suggestion
   mysql -h cluster1-proxysql -uroot -proot_password
```

File: source/kubernetes.rst [hunk 2]
```diff
@@ -92,15 +92,35 @@ Install Percona XtraDB Cluster on Kubernetes
...
       percona-xtradb-cluster-operator-dc67778fd-qtspz   1/1     Running   0          6m
 
-7. Check connectivity to newly created cluster
+#. Check connectivity to newly created cluster
 
    .. code:: bash
 
       $ kubectl run -i --rm --tty percona-client --image=percona:5.7 --restart=Never -- bash -il
-      percona-client:/$ mysql -h cluster1-proxysql -uroot -proot_password
+      percona-client:/$ mysql -h cluster1-haproxy-0 -uroot -proot_password
```
Comment (delgod) [inline source/kubernetes.rst:108]: let's connect via service
```suggestion
      percona-client:/$ mysql -h cluster1-haproxy -uroot -proot_password
```

File: source/minikube.rst [hunk 3]
```diff
@@ -83,4 +98,22 @@ The following steps are needed to run PXC Operator on Minikube:
    
    .. code:: bash
 
-      mysql -h cluster1-proxysql -uroot -proot_password
+      mysql -h cluster1-haproxy-0 -uroot -proot_password
```
Comment (delgod) [inline source/minikube.rst:101]:
```suggestion
      mysql -h cluster1-haproxy -uroot -proot_password
```

File: source/openshift.rst [hunk 4]
```diff
@@ -103,15 +103,35 @@ Install Percona XtraDB Cluster on OpenShift
...
       percona-xtradb-cluster-operator-dc67778fd-qtspz   1/1     Running   0          6m
 
-7. Check connectivity to newly created cluster
+#. Check connectivity to newly created cluster
 
    .. code:: bash
 
       $ oc run -i --rm --tty percona-client --image=percona:5.7 --restart=Never -- bash -il
-      percona-client:/$ mysql -h cluster1-proxysql -uroot -proot_password
+      percona-client:/$ mysql -h cluster1-haproxy-0 -uroot -proot_password
```
Comment (delgod) [inline source/openshift.rst:119]:
```suggestion
      percona-client:/$ mysql -h cluster1-haproxy -uroot -proot_password
```

**Review by delgod — CHANGES_REQUESTED** [review]

File: source/gke.rst [hunk 5]
```diff
@@ -176,11 +176,11 @@ This command will connect you to the MySQL monitor.
 
    mysql: [Warning] Using a password on the command line interface can be insecure.
    Welcome to the MySQL monitor.  Commands end with ; or \g.
-   Your MySQL connection id is 111
-   Server version: 5.5.30 (ProxySQL)
+   Your MySQL connection id is 119
+   Server version: 8.0.19 (ProxySQL)
```
Comment (delgod) [inline source/gke.rst:180]: should we rework it for HAProxy?

**Review by delgod — CHANGES_REQUESTED** [review]

File: source/gke.rst (hunk 5 above)
Comment (fiowro) [reply source/gke.rst:180]: I think it's better to have at least one ProxySQL example somewhere

**Review by delgod — APPROVED** [review]

File: source/gke.rst (hunk 5 above)
Comment (fiowro) [reply source/gke.rst]: done

**Review by delgod — APPROVED** [review]

**Pull request merged**